packages/core-contracts/contracts/ownership/Ownable.sol

```diff
@@ -13,11 +13,15 @@ contract Ownable is IOwnable, OwnableMixin {
     error NotNominated(address addr);
 
     function acceptOwnership() external override {
+        _acceptOwnershipBy(msg.sender);
```

Comment:
added the internal function to accept ownership from the initialization function